tools/tools.go

```diff
@@ -1,7 +1,9 @@
+//go:build tools
 // +build tools
 
 package tools
 
 import (
 	_ "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs"
+	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
```

Comment:
Do we need this here if we're running the lint command directly from GH?

Reply:
The GH action just runs `make lint`, which originally just `curl`ed the binary. I switched it up to use `go run` and added it here as a dependency, which seemed more consistent.

Reply:
I'm just not sure if development only dependency should be included here (or go.mod)